Docs/source/running_cpp/parameters.rst

```diff
@@ -276,7 +276,7 @@ Particle initialization
       ``<species_name>.x/y/z_m`` (average position in `x/y/z`),
       ``<species_name>.x/y/z_rms`` (standard deviation in `x/y/z`),
       ``<species_name>.x/y/z_rms`` (standard deviation in `x/y/z`),
-      ``<species_name>.x/y/z_cut`` (optional, particles with ``abs(x-x_m) > x_cut*x_rms`` are not injected, same for y and z),
+      ``<species_name>.x/y/z_cut`` (optional, particles with ``abs(x-x_m) > x_cut*x_rms`` are not injected, same for y and z. Note that cutting the distribution is likely to result in a lower total charge),
```

Comment:
This is a very good point @MaxThevenet !
I don't know how to explain it. But I would make sure users still initialize the beam with the correct `q_tot`.
Maybe something like this:
```suggestion
      ``<species_name>.x/y/z_cut`` (optional, particles with ``abs(x-x_m) > x_cut*x_rms`` are not injected, same for y and z. Note that cutting the distribution is likely to result in a lower total charge of macro-particles in your output diagnostics. But the `<species_name>.q_tot` parameter should still be the total charge of the particles (without cut).),
```